packages/app/src/cli/services/extensions/common.ts

```diff
@@ -49,28 +47,13 @@ export async function ensureExtensionDirectoryExists({name, app}: {name: string;
   return extensionDirectory
 }
 
-export async function renderDevPreviewWarning(
-  remoteApp: Partial<OrganizationApp>,
-  localApp: AppInterface,
-): Promise<void> {
-  const body = await buildDevPreviewWarning(remoteApp, localApp)
-  if (!body) return
-
-  outputInfo(`\n${body}\n`)
-}
-
-export async function buildDevPreviewWarning(remoteApp: Partial<OrganizationApp>, localApp: AppInterface) {
+export function isPreviewModeEnabled(remoteApp: Partial<OrganizationApp>, localApp: AppInterface) {
   const unifiedDeployment = remoteApp?.betas?.unifiedAppDeployment ?? false
-  if (!unifiedDeployment) return
+  if (!unifiedDeployment) return false
```

Comment:
Let's ignore this suggestion for now. Once we go through the codebase to remove code related to unified deployments we can remove this check too.

Reply:
@matteodepalo what we are doing here is not displaying the `dev preview` toggle shortcut in the footer in case `unified deployments` is not enabled. I think that in that case the `dev preview mode` does not take any effect on how devs preview their changes right?. `q` and `p` shortcuts are always available regardless the beta flag

Reply:
But in case you don't opt into `unified deployments` and you have a `theme app extension` I don't know if the new API request will work. I will check that and came back to you with the feedback

Reply:
@matteodepalo you are totally right!, it's possible to use the API and add the new shortcut so it's easier for app with theme app extension that has not opted into `unified deployments`